README.md

```diff
@@ -32,6 +32,11 @@ If you need to install Go, follow [the official instructions](https://golang.org
 
 Find up-to-date examples and documentation on [godoc.org](https://godoc.org/github.com/Azure/azure-service-bus-go#pkg-examples).
 
+### Running tests
+
+Most tests require a properly configured service bus in Azure.  The easiest way to set this up is to use the [Terraform](https://www.terraform.io/) deployment script.
```

Comment:
I would just put the name of the file azuredeploy.tf here, unless we're going to explain how to run the tests with the makefile, then definitely just leave the makefile test information.